run.py

```diff
@@ -81,7 +81,8 @@ def create_database(db_connection, db_schema):
 
         if not session().query(schemata_exists).scalar():
             logger.info("Creating schema ", db_schema=db_schema)
-            engine.execute(f"CREATE SCHEMA {db_schema}")
+            session().execute(text(f"CREATE SCHEMA {db_schema}"))
```

Comment:
2.0 doesn't let `engine.execute` anymore.  You have to do it either through a session, like the one we've created already, or through a connection.